### PR TITLE
[webview_flutter] Apply PlatformView API change

### PR DESCRIPTION
**Pull request by bbrto21**
Signed-off-by: Boram Bae <boram21.bae@samsung.com>

### CODE REVIEW

**Comment by bbrto21** [comment]
this PR depends on https://github.com/flutter-tizen/engine/pull/291 and it requires engine hash update on flutter-tizen

File: packages/webview_flutter/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## 0.4.5
+* Apply PlatformView API change.
+
 ## NEXT
 
 * Code refactoring.
```
Comment (swift-kim) [inline packages/webview_flutter/CHANGELOG.md:6]:
```suggestion
## 0.4.5

* Apply PlatformView API change.
* Code refactoring.
```

Comment (bbrto21) [reply packages/webview_flutter/CHANGELOG.md:6]: Ok, I'll update this PR after https://github.com/flutter-tizen/plugins/pull/397 gets merged

File: packages/webview_flutter/tizen/src/webview.cc [hunk 2]
```diff
@@ -518,128 +518,112 @@ static LWE::KeyValue KeyNameToKeyValue(const std::string& ecore_key,
...
+          param->webview_instance->DispatchKeyDownEvent(param->key_value);
+          param->webview_instance->DispatchKeyPressEvent(param->key_value);
+        } else {
+          param->webview_instance->DispatchKeyUpEvent(param->key_value);
+        }
         delete param;
       },
       param);
+
+  return false;
```
Comment (swift-kim) [inline packages/webview_flutter/tizen/src/webview.cc:626]: I think we've discussed about adding a `CanGoForward` check on a back key press and invoking `GoBack` if the result is true. Are you going to implement that later?

Comment (swift-kim) [reply packages/webview_flutter/tizen/src/webview.cc:626]: Thanks!

**Review by swift-kim — APPROVED** [review]

**Comment by bbrto21** [comment]
@HakkyuKim Is there something wrong with the integration_test action?

**Pull request merged**